python/ray/serve/controller.py

```diff
@@ -523,13 +532,7 @@ async def __init__(self,
                        detached: bool = False):
         # Used to read/write checkpoints.
         self.kv_store = RayInternalKVStore(namespace=controller_name)
-        # Current State
-        self.current_state = SystemState()
-        # Goal State
-        # TODO(ilr) This is currently *unused* until the refactor of the serve
-        # controller.
-        self.goal_state = SystemState()
-        # ActorStateReconciler
+        self.backend_state = BackendState()
```

Comment:
Can you delete this? it is done below (line 565)

Reply:
oops, will do